tests/unit/test_transfer_manager.py

```diff
@@ -242,15 +248,15 @@ def test_download_many_with_filenames():
     ]
 
     for blob_mock, _ in BLOB_FILE_PAIRS:
-        blob_mock.download_to_filename.return_value = FAKE_RESULT
+        blob_mock._handle_filename_and_download.return_value = FAKE_RESULT
 
     results = transfer_manager.download_many(
         BLOB_FILE_PAIRS,
         download_kwargs=DOWNLOAD_KWARGS,
         worker_type=transfer_manager.THREAD,
     )
     for (mock_blob, file) in BLOB_FILE_PAIRS:
-        mock_blob.download_to_filename.assert_any_call(file, **DOWNLOAD_KWARGS)
+        mock_blob._handle_filename_and_download.assert_any_call(file, **DOWNLOAD_KWARGS)
```

Comment:
Similar for the downloads tests... Instead of adding `"command": "tm.download_many"` to `DOWNLOAD_KWARGS`, assert against a newly created `EXPECTED_DOWNLOAD_KWARGS` that includes the command